tests/G2SubGroupCheckTests.cry

```diff
@@ -5,32 +5,38 @@ import ShortWeierstrassCurve
 import G2SubGroupCheck
 
 property test_psi_1 = is_point_affine E' (psi BP') // psi takes BP' to a point on the curve
-
-property test_psi_2 (m:[16]) = is_point_affine E' (psi (mult E' m BP')) // and multiples of BP'
+// and multiples of BP':
+test_psi_2 m = is_point_affine E' (psi (mult E' m BP')) // and multiples of BP'
```

Comment:
Another option for testing less is to put `:set tests=2`in the checking script.  But an advantage of what you have here is determinism and repeatability.